src/main/java/com/example/TicketBookingApp/repository/ReservationRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.example.TicketBookingApp.repository;
+
+import com.example.TicketBookingApp.domain.Reservation;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface ReservationRepository extends JpaRepository<Reservation, Long> {
+    Optional<Reservation> findById(Long id);
```

Comment:
plus za optionala

Reply:
Minut za to, że metoda znajduje się w JpaRepository :D